version_control/katalyst_by_codurance/css/elements/_typography.css

```diff
@@ -78,11 +78,18 @@ code {
 }
 
 /* Blockquotes */
+blockquote:before {
+  content: open-quote;
+  font-size: var(--fs-4);
+  line-height: 1;
+}
+
+blockquote p {
+  font-size: var(--fs-1);
```

Comment:
Even though I've seen that with the copy from the design this font-size was a good choice, it seems that the final copy is much larger and the cards look really large 😢 

![image](https://github.com/codurance/katalyst-by-codurance-microsite/assets/44123032/7ddec5af-bb17-4692-87c1-70252a569e02)

I've tried with the --fs-0 size and that way seems to be just fine. What do you think?

Reply:
Changed! I was doubting on this one as well...